tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -134,7 +135,7 @@ export enum CardFeatureOverride {
 
 export type CardState = {
   dataMinMax: MinMaxStep;
-  userMinMax: MinMaxStep;
+  userViewBox: Extent;
```

Comment:
Do we also need to store the xAxisType of the card (STEP, WALL, RELATIVE)? userViewBox does not seem like the entire story. I think xAxisType didn't really matter before because userMinMax was only ever read when xAxistType was STEP and this was enforced at the component level.

(Doesn't need to change in this PR but perhaps you add it in subsequent PRs.)

Reply:
Thanks for bring this up. I have my doubt that xAxisType matters. Different xAxisType would mean the numbers are different but once rehydrated it will come with the xAxisType.
I will make note here to see if xAxisType is needed in userViewBox.